### PR TITLE
Update add-storage-adapter.md

### PR DESCRIPTION
**Pull request by tusharxoxoxo**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

**Comment by tessamero** [comment]
"constant" should remain constant as it makes more sense not to use plural

"may be" is correct and shouldn't be changed to "maybe". I had to read it out loud a few times. It's the sentence structure that needs to be rewritten to make more sense :)

File: docs/tutorials/add-storage-adapter.md [hunk 1]
```diff
@@ -8,20 +8,20 @@ This document is part of the Appwrite contributors' guide. Before you continue r
...
-Add a new storage adapter inside `src/Storage/Device/` folder. Use one of the existing ones as a reference. The new adapter class should extend `Device` class and implement all the required methods.
+Add a new storage adapter inside the `src/Storage/Device/` folder. Use one of the existing ones as a reference. The new adapter class should extend `Device` class and implement all the required methods.
 
 Note that the class name should start with a capital letter as PHP FIG standards suggest.
 
 Always use properly named environment variables if any credentials are required.
 
 ### Introduce new device constant
-Introduce newly added device constant in `src/Storage/Storage.php` alongside existing device constants. The device constant should start with `const DEVICE_<name of device>` as the existing ones.
+Introduce newly added device constants in `src/Storage/Storage.php` alongside existing device constants. The device constant should start with `const DEVICE_<name of device>` as the existing ones.
```
Comment (gewenyu99) [inline docs/tutorials/add-storage-adapter.md:24]: Please revert this change.

File: docs/tutorials/add-storage-adapter.md [hunk 2]
```diff
@@ -34,18 +34,18 @@ If everything goes well, create a new pull request in [utopia-php/storage](https
...
 
-* Note for this to happen, your PR in the first phase should have been merged and new version of [utopia-php/storage](https://github.com/utopia-php/storage) library released.
+* Note for this to happen, your PR in the first phase should have been merged and a new version of [utopia-php/storage](https://github.com/utopia-php/storage) library released.
 
 ### Upgrade the utopia-php/storage dependency
 Upgrade the utopia-php/storage dependency in `composer.json` file.
 
 ### Introduce new environment variables
-If required for the new adapter, may be for credentials, introduce new environment variables. The storage environment variables are prefixed as `_APP_STORAGE_DEVICE`. Please read [Adding Environment Variables]() guidelines in order to properly introduce new environment variables.
+If required for the new adapter, maybe for credentials, introduce new environment variables. The storage environment variables are prefixed as `_APP_STORAGE_DEVICE`. Please read [Adding Environment Variables]() guidelines in order to properly introduce new environment variables.
```
Comment (gewenyu99) [inline docs/tutorials/add-storage-adapter.md:43]: Suggested phrasing:
```
Introduce new environment variables if the adaptor requires new configuration information or to pass in credentials. The storage environment variables are prefixed as `_APP_STORAGE_DEVICE`. Please read [Adding Environment Variables]() guidelines in order to properly introduce new environment variables.
```

Notice how `[Adding Environment Variables]()` is also broken? This isn't your fault, but it would be great if you can fix this, too.

**Review by gewenyu99 — DISMISSED** [review]

File: docs/tutorials/add-storage-adapter.md (hunk 1 above)
Comment (tusharxoxoxo) [reply docs/tutorials/add-storage-adapter.md]: Done

File: docs/tutorials/add-storage-adapter.md (hunk 2 above)
Comment (tusharxoxoxo) [reply docs/tutorials/add-storage-adapter.md]: is it correct now??

**Comment by Meldiron** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Comment by stnguyen90** [comment]
@tusharxoxoxo would you please fix the merge conflicts and then re-request a review?

File: docs/tutorials/add-storage-adapter.md (hunk 2 above)
Comment (christyjacob4) [reply docs/tutorials/add-storage-adapter.md]: I have updated it

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)
